package/README.md

```diff
@@ -0,0 +1,178 @@
+To ensure reliability, availability and neutrality it is desirable that those servers are
+being operated by multiple independent entities.
+
+Therefore we provide this repository which allows easy setup of these services.
+It uses docker and docker-compose for easy installation and upgrades.
+
+### Used software
+
+- docker
+- docker-compose
+- Traefik
+
+### Structure
+
+
+```
++-------------------+
+|                   |
+|   Raiden clients  |
+|                   |
++---------+---------+
+          |
+==========|==========
+          |
++---------v---------+
+|                   |
+|      Traefik      |
+|                   |
++---------+---------+
+          |
+          +------------------------+
+          |                        |
++---------v---------+    +---------v---------+
+|                   |    |                   |
+|     Pathfinding   |    |    Monitoring     |
+|                   |    |                   |
++---------+---------+    +---------+---------+
+```
+
+
+We use Traefik as a reverse proxy and also utilize it's capability of automatically provisiong
+Let's Encrypt TLS certificates.
+
+### Network
+
+After a successful deployment the following ports will be in use:
+
+- 80 - HTTP
+  - Redirects to HTTPS
+  - Let's Encrypt HTTP challenge for certificate provisioning
+- 443 - HTTPS
+  - Synapse web and API client access
+  - Metrics export (IP restricted, see below)
+ 
+TODO: check
+
+## Requirements
+
+### Hardware
+
+Minumum recommended for a production setup:
+
+- 2 GiB Ram
+- 2 Cores
+- 10 GiB SSD
+
+### Software
+
+- Docker >= 17.12
+- docker-compose >= 1.21.0
+
+### Other
+
+- A domain (or subdomain) for exclusive use by the services
+- A synced Ethereum node, that is reachable from the server where the services are installed,
+is required. Setting one up is outside of the scope of this document, please refer to
+ <some link to eth node setup instructions>.
+- A fresh Ethereum account that will be used only by the services. The accounts needs a small
+amount of funding (0.1 ETH) should be enough.
+
+## Installation
+
+### Preparation
+
+1. Provision a server that meets the [hardware](#hardware) and [software](#software) requirements listed above.
+1. Ensure a domain (or subdomain) is available
+
+   Examples:
+   - raidenservices.mydomain.tld
+   - raidenservices-mydomain.tld
+   - mydomain-raidenservices.tld
+
+1. Configure `A` (and optionally `AAAA`) DNS records for the domain pointing to the servers IP address(es)
+1. Configure a `CNAME` DNS record for `*.<domain>` pointing back to `<domain>`
+
+TODO: check
+
+### Installing
+
+1. Clone the [current release version of this repository](https://github.com/raiden-network/raiden-services)
+   to a suitable location on the server:
+
+   ```shell
+   git clone https://github.com/raiden-network/raiden-services.git
+   ```
+1. Copy the keystore file to be used by the services into the directory `./data/keystore/`
```

Comment:
I have problems with permissions using that folder. It works but it is not obvious to copy with sudo and change permissions after that. Is that something we can change?

Reply:
It's strange that your keystore file was written by root. But I'll add a hint that the docker container needs permissions to read that file.

Reply:
yes but it also happened the second time, so maybe that is a ubuntu problem